popup.js

```diff
@@ -151,32 +176,45 @@ modal2.insertAdjacentHTML(
         <button class='closepopup'>&times;</button>
     </nav>
   <ul class="offer-pop">
-  <li class="canopy">${containersworks[0].offer[0]}</li>
+  <li class="canopy">${containersworks[2].offer[0]}</li>
   <span class="point"></span>
-  <li class="backend">${containersworks[1].offer[1]}</li>
+  <li class="backend">${containersworks[2].offer[1]}</li>
   <span class="point"></span>
   <li class="year">${containersworks[2].offer[2]}</li>
 </ul>
-    <img class="snapshootpop" src="Images/Snapshoot3.png" alt="mobile image">
+    <img class="snapshootpop" src="${containersworks[2].snapshoot}" alt="mobile image">
     <img class="desktoppop" src="${containersworks[2].desktop}" alt="Desktop Image">
  
-    <div class="daily">
-       <p class="description-pop">${containersworks[0].description}</p>
-       
-       <ul class="skills">
-       <li class="htl"><a href="#"></a>${containersworks[0].skills[0]}</li>
-       <li class="css"><a href="#"></a>${containersworks[1].skills[1]}</li>
+    <div class="dailypop">
+       <p class="description-pop">${containersworks[2].description}</p>
+        <p class="description-pop2">${containersworks[2].descriptionpop}</p>
+       <div>
+       <ul class="skillspop">
+       <li class="htl"><a href="#"></a>${containersworks[2].skills[0]}</li>
+       <li class="css"><a href="#"></a>${containersworks[2].skills[1]}</li>
+       <li class="js"><a href="#"></a>${containersworks[2].skills[2]}</li>
+       </ul>
+
+       <ul class="skillspop2">
+       <li class="htl"><a href="#"></a>${containersworks[2].skills[0]}</li>
+       <li class="css"><a href="#"></a>${containersworks[2].skills[1]}</li>
        <li class="js"><a href="#"></a>${containersworks[2].skills[2]}</li>
+        </ul>
+       <ul class="skillspop2">
+       <li class="css"><a href="#"></a>${containersworks[2].skills[3]}</li>
+       <li class="js"><a href="#"></a>${containersworks[2].skills[4]}</li>
```

Comment:
- [ ] Great job on using array object for the values of each popup container however to make sure your code is `DRY`  I would suggest using loops to iterate over the object array instead of manually creating each popup from the array object.  For more information on best practices read [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md). This will not only: 
- [ ] make adding projects in the future a breeze. As you will only need to add project details into your array object and your javascript will create the popup for your new project instead of you putting it manually. 
- [ ] It will also make your code more clean and manageable

Reply:
Creating the objects and manipulating them using javascript, it's also good practice.